https/testdata/tls_config_junk.yml

```diff
@@ -0,0 +1,27 @@
+I wandered lonely as a cloud
```

Comment:
While this is well out of copyright, it'd be best to use actual junk so companies using this don't have to send lawyers to verify.